src/pages/on-chain-governance/_components/ConfirmVoteDialog.tsx

```diff
@@ -477,3 +601,42 @@ function ReadyVoteID({
     </>
   );
 }
+
+function getVotesStyle(vote: VoteDecision | undefined) {
+  if (vote === VoteDecision.NO) {
+    return "text-red-600";
+  }
+  if (vote === VoteDecision.YES) {
+    return "text-green-600";
+  }
+  if (vote === VoteDecision.NEUTRAL) {
+    return "text-gray-600";
+  }
+}
+
+function setMasternodeOptions(
+  connection: NetworkConnection,
+  rememberMasternodeId: RememberMasterNodeId,
+  masternodeId: string,
+  setVoteStage: Dispatch<SetStateAction<VoteStages>>
+) {
+  // retrieve object that is stored in localStore
```

Comment:
move function description on top of function declaration, replace with multiline comment